ovos_utils/gui.py

```diff
@@ -827,6 +827,7 @@ def show_image(self, url, caption=None,
             title (str): A title to display above the image content
             fill (str): Fill type supports 'PreserveAspectFit',
             'PreserveAspectCrop', 'Stretch'
+            background_color (str): A background color for the page
```

Comment:
Docstring should specify format (RGB hex code?) Maybe just an example would suffice `(i.e. #000000)`